pkg/apis/pipeline/v1/taskrun_validation_test.go

```diff
@@ -111,6 +131,28 @@ func TestTaskRun_Validate(t *testing.T) {
 			},
 		},
 		wc: config.EnableAlphaAPIFields,
+	}, {
+		name: "propagating partial params with different provided and default names",
```

Comment:
this test case has the same name as the next one, can you make one or both more descriptive?

Reply:
Sure.